app/containers/MessageActions/index.js

```diff
@@ -171,7 +171,7 @@ const MessageActions = React.memo(forwardRef(({
 
 	const handleCopy = async(message) => {
 		logEvent(events.ROOM_MSG_ACTION_COPY);
-		await Clipboard.setString(message.msg);
+		await Clipboard.setString(message.attachments.length > 0 ? message.attachments[0].description : message.msg);
```

Comment:
Can you simplify with something like `message?.attachments?.[0]?.description || message.msg`?